src/main/scala/cognite/spark/v1/FlexibleDataModelBaseRelation.scala

```diff
@@ -56,36 +56,65 @@ abstract class FlexibleDataModelBaseRelation(config: RelationConfig, sqlContext:
     })
   }
 
+  // scalastyle:off method.length
   // scalastyle:off cyclomatic.complexity
-  private def extractInstancePropertyValue: InstancePropertyValue => Any = {
```

Comment:
A bit unsure what changes and why we'll no longer need typing info/view with this change.
Before it was doing FDM API values -> plain values conversion based on their type.
Now it still uses type but adds more casts (to fit spark schema better?) and looks into spark schema.
So there's no change around looking into FDM or view? On itself of course looks safe, just not quite sure how it connects to the sync endpoint work

Reply:
there are two paths in the sparc sdks for deserialising the json into 
InstancePropertyValue.Int32 or InstancePropertyValue.Int64. One path is if you fetch down the view definition inline in the request, then the deserialiser will look at the view and resolve correct type. Otherwise it will select one base on the shape of the data, and can end up with any of int, long, float or double..

Reply:
Ah, ok, so sdk call parameter change is the goal but then we need to do some type conversions :+1: